client.py

```diff
@@ -0,0 +1,94 @@
+"""
+Implement a tkinter client who communicates with a fastapi server
+and executes commands
+"""
+
+import requests
+from threading import Thread
+import tkinter
+
+
+root = tkinter.Tk()
+frame = tkinter.Frame(root)
+command_inp = tkinter.Text(frame, width=70, height=1, fg="green")
+
+
+def retrieve_input():
+    """
+    Get a command input from the user and get the output to the command
+    from a fastapi server
+    """
```

Comment:
Re-phrase to something like 
```suggestion
    """ Get use input and send relevant command to the server """
```